files/cluster_provision.yml

```diff
@@ -66,34 +66,41 @@
                                 kernel_opts="ip=dhcp text inst.stage2=http://releases.linaro.org/reference-platform/enterprise/18.06/centos-installer/ inst.repo=http://mirror.centos.org/altarch/7/os/aarch64/ inst.ks=file:/ks.cfg"
                         fi
 
-                        # Chose known good kernel/initrd/cmdline
-
-                        # Build trigger machine_provision job
-                        cat << EOF > machine_provision
-                        machine_list=${machine_list}
-                        os_type=${os_type}
-                        machine_type=${machine_type}
+                        # Defaults
+                        machine_list="$( echo "$machine_list"| tr -d '[:space:]')"
+                        machine_name=${machine_list}
+                        machine_arch=AArch64
+                        machine_subarch=Grub
                         job_type=ohpc
+
+                        kernel_opts="${kernel_opts} earlycon console=ttyAMA0,115200"
+
+                        # Special cases
+                        if [ "${machine_type}" == "qdc" ]; then
```

Comment:
Why not set this up there, where you set the qualcomm parameters?

Reply:
d05 uses it too !

Reply:
`machine_subarch` is different for d03/d05 and qdf, possible even tx2, so it could be set with the other arguments in the if/else above.